tests/test_registrations/test_views.py

```diff
@@ -40,6 +40,16 @@ def test_node_register_page_registration(self, mock_archive):
         res = self.app.get(url, auth=self.user.auth)
         assert_equal(res.status_code, http.OK)
 
+    @mock.patch('website.archiver.tasks.archive')
+    def test_node_register_over_100_nodes(self, mock_archive):
```

Comment:
This test takes ~30 seconds to run, and doesn't fail on `develop` without the changes to `register_node`, meaning that it's not useful as a regression test. Please ensure this test can fail.

```
~/D/osf.io ❯ g diff                                    remotes/cos/develop 
diff --git a/tests/test_registrations/test_views.py b/tests/test_registrations/test_views.py
index b932585..1513710 100644
--- a/tests/test_registrations/test_views.py
+++ b/tests/test_registrations/test_views.py
@@ -28,6 +28,16 @@ from tests.base import get_default_metaschema

 class TestRegistrationViews(RegistrationsTestBase):

+    @mock.patch('website.archiver.tasks.archive')
+    def test_node_register_over_100_nodes(self, mock_archive):
+        for i in xrange(110):
+            node = NodeFactory(parent=self.node)
+            node.save()
+        reg = self.node.register_node(get_default_metaschema(), self.auth, '', None)
+        url = reg.web_url_for('node_register_page')
+        res = self.app.get(url, auth=self.user.auth)
+        assert_equal(res.status_code, http.OK)
+
     def test_node_register_page_not_registration_redirects(self):
         url = self.node.web_url_for('node_register_page')
         res = self.app.get(url, auth=self.user.auth)
```
```
~/D/osf.io ❯ nosetests tests/test_registrations/test_views.py:TestRegistrationViews.test_node_register_over_100_nodes         remotes/cos/develop
[logging omitted...]
.
----------------------------------------------------------------------
Ran 1 test in 30.840s

OK
```